src/core/notifier/handler/notification/processor.go

```diff
@@ -172,3 +173,30 @@ func preprocessAndSendImageHook(value interface{}) error {
 	return nil
 
 }
+
+// will return nil when it failed to get data
+func getScanOverview(digest string, tag string, eventType string) *models.ImgScanOverview {
+	if len(digest) == 0 {
+		log.Debug("digest is nil")
+		return nil
+	}
+	data, err := dao.GetImgScanOverview(digest)
+	if err != nil {
+		log.Errorf("Failed to get scan result for tag:%s, digest: %s, error: %v", tag, digest, err)
```

Comment:
return err?

Reply:
i pick the first one between missing some part of payload and dropping the triggered event.